tbx/work/models.py

```diff
@@ -269,6 +273,21 @@ def read_time(self):
         else:
             return "x"
 
+    @property
+    def listing_image(self):
+        return self.header_image
+
+    @property
+    def work_index(self):
```

Comment:
I had to add this back, as we can't use a relative link for the tag link base on service pages.